pkgs/applications/virtualization/virtualbox/default.nix

```diff
@@ -76,11 +76,12 @@ in stdenv.mkDerivation {
      optional enableHardening ./hardened.patch
   ++ [
     ./qtx11extras.patch
-    (fetchpatch {
-      name = "010-qt-5.11.patch";
-      url = "https://git.archlinux.org/svntogit/community.git/plain/trunk/010-qt-5.11.patch?h=packages/virtualbox";
-      sha256 = "0hjx99pg40wqyggnrpylrp5zngva4xrnk7r90i0ynrqc7n84g9pn";
-    })
+    # https://www.virtualbox.org/ticket/18620
+    ./fix_kbuild.patch
```

Comment:
They're very small and we have other patches right here in nixpkgs, so I think it doesn't make sense.

Reply:
Nixpkgs policy is to not include patches when possible. Many of the patches in Nixpkgs are added under the same "well there are other patches in here...", but it is generally wrong. When possible (like in most of these cases) the patches should be included via fetchpatch.

Reply:
To clarify, I'm grateful of your work here -- this PR is in great shape, despite these patches being checked in. In the future, please fetch patches (and @flokli please be a bit more insistent :) )

Reply:
And, since I managed to not say it in the first two messages (d'oh):

Thank you _so much_ for your work in Nixpkgs, patching, and working with upstream to get this going. I really appreciate it, it makes a big difference to NixOS!